api/RTCPeerConnection.json

```diff
@@ -0,0 +1,3237 @@
+              "version_added": "51",
+              "version_removed": "56"
+            },
+            {
+              "notes": "unprefixed",
+              "version_added": "56"
+            }
+          ],
+          "chrome_android": [
+            {
+              "prefix": "webkit",
+              "version_added": "51",
+              "version_removed": "56"
+            },
+            {
+              "notes": "unprefixed",
+              "version_added": "56"
+            }
+          ],
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "22"
+          },
+          "firefox_android": [
+            {
+              "prefix": "moz",
+              "version_added": "22",
+              "version_removed": "44"
+            },
+            {
+              "notes": "unprefixed",
+              "version_added": "44"
+            }
+          ],
+          "ie": {
+            "version_added": null
+          },
+          "opera": [
+            {
+              "version_added": "37",
+              "version_removed": "43"
+            },
+            {
+              "version_added": "43",
+              "notes": "Promise based version."
+            }
+          ],
+          "opera_android": [
+            {
+              "version_added": "37",
+              "version_removed": "43"
+            },
+            {
+              "version_added": "43",
+              "notes": "Promise based version."
+            }
+          ],
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "RTCPeerConnection": {
+        "__compat": {
+          "description": "RTCPeerConnection()",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/RTCPeerConnection",
+          "support": {
+            "webview_android": {
+              "version_added": "56"
+            },
+            "chrome": {
+              "version_added": "56"
+            },
+            "chrome_android": {
+              "version_added": "56"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "22"
+            },
+            "firefox_android": {
+              "version_added": "44"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": [
+              {
+                "version_added": "37",
+                "version_removed": "43"
+              },
+              {
+                "version_added": "43",
```

Comment:
Here as well. The most relevant version needs to come first. Swap this around please:
```json
              {
                "version_added": "43",
                "notes": "Promise based version."
              },
              {
                "version_added": "37",
                "version_removed": "43"
              }
```

Reply:
I'm on it..